package.json

```diff
@@ -3,10 +3,14 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
+    "build": "node --experimental-modules build.mjs && npm run build-sass && npm run build-js",
+    "build-sass": "node-sass stylesheets/style.scss public/stylesheets/style.css --output-style compressed --source-map true",
+    "build-js": "npm run rollup && npm run transpile-js",
+    "dev": "nodemon -e js,hbs,scss --ignore public/javascript --exec 'npm run build && DEBUG=:* & npm start'",
+    "rollup": "rollup -c",
     "start": "node ./bin/www",
-    "build": "node --experimental-modules build.mjs",
-    "dev": "nodemon -e js,hbs,scss --ignore public/javascripts --exec 'npm run build && DEBUG=:* & npm start'",
-    "test": "npx jest --verbose tests/javascripts/**/*.test.js"
+    "test": "npx jest --verbose tests/javascript/**/*.test.js",
```

Comment:
The actual test folder will also need renaming

Reply:
ah good spot